vnext/ReactUWP/Utils/ValueUtils.cpp

```diff
@@ -49,13 +50,51 @@ REACTWINDOWS_API_(winrt::Color) ColorFrom(const folly::dynamic& d)
 
 REACTWINDOWS_API_(winrt::SolidColorBrush) SolidColorBrushFrom(const folly::dynamic& d)
 {
-  thread_local static std::map<winrt::Color, winrt::weak_ref<winrt::SolidColorBrush>, ColorComp> solidColorBrushCache;
+  if (d.isObject())
+  {
+    winrt::hstring resourceName{ winrt::to_hstring(d.find("windowsbrush")->second.asString()) };
+
+    thread_local static std::map<winrt::hstring, winrt::weak_ref<winrt::SolidColorBrush>> accentColorMap = {
```

Comment:
>accentColorMap [](start = 90, length = 14)

Do you need to refresh the map when theme is changed?

Reply:
No, the ThemeResource keyword takes care of that.